src/solvers/floatbv/float_utils.cpp

```diff
@@ -136,6 +136,32 @@ bvt float_utilst::to_integer(
   return result;
 }
 
+bvt float_utilst::round_to_integral(const bvt &src)
```

Comment:
Where does this actually consider the rounding mode?

Reply:
It calls `add_sub`.